dojo/views.py

```diff
@@ -99,6 +99,7 @@ def action_history(request, cid, oid):
                   {"history": paged_history,
                    'product_tab': product_tab,
                    "filtered": history,
+                   "log_entry_filter": log_entry_filter,
```

Comment:
is this used somewhere?

Reply:
The `log_entry_filter` was unused, but I activated it in the history view in `action_history.html`.